csrc/scheduler/normalization_inner_outer_tma_ws.cpp

```diff
@@ -281,13 +191,14 @@ void getHeuristics(
   rparams->vectorize_inner_reduction = iop.inner_vect > 1;
   rparams->split_grid_dim_iter_dom_outer = true;
   rparams->grid_dim_iter_dom = ParallelType::BIDy;
+  rparams->pad_inner_reduction_to_warp = true;
 
   rparams->lparams = LaunchParams(
       LaunchParams::UNINITIALIZED_VAL,
       iop.gdimy,
       LaunchParams::UNINITIALIZED_VAL,
       iop.bdimx,
-      iop.bdimy,
+      LaunchParams::UNINITIALIZED_VAL,
```

Comment:
naive question, we are still using bdimy for warp specialization, do we not need to specify a launch config for that?

Reply:
That is being handled in `void ParallelDimensionMap::adjustMappingsForWarpSpecialization()`